### PR TITLE
decompile field_message_box

### PR DESCRIPTION
**Pull request by golemgalvanize**

### CODE REVIEW

File: src/field_message_box.c [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+extern void sub_81973A4(void);
+extern void sub_81973C4(u8, u8);
+extern u16 sub_8197224(void);
+extern int sub_8196094(void);
+extern void sub_8196080(u8*);
+extern void AddTextPrinterForMessage(u8);
+extern void sub_8197434(u8, u8);
+extern void sub_81973FC(u8, u8);
+
+extern u8 gUnknown_020375BC;
```
Comment (Diegoisawesome) [inline src/field_message_box.c]: Need to insert this in EWRAM

File: src/field_message_box.c [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+#include "text.h"
+
+extern void sub_81973A4(void);
+extern void sub_81973C4(u8, u8);
+extern u16 sub_8197224(void);
+extern int sub_8196094(void);
+extern void sub_8196080(u8*);
+extern void AddTextPrinterForMessage(u8);
+extern void sub_8197434(u8, u8);
+extern void sub_81973FC(u8, u8);
```
Comment (Diegoisawesome) [inline src/field_message_box.c]: All these are either already in new_menu_helpers.h or should be (yes, even the Battle Frontier ones, it was a bad split)

Comment (golemgalvanize) [reply src/field_message_box.c]: just the last four or all of the functions

Comment (Diegoisawesome) [reply src/field_message_box.c]: All of them, I couldn't find any that weren't in new_menu_helpers or battle_frontier_1

File: sym_ewram.txt [hunk 3]
```diff
@@ -643,7 +643,6 @@ gUnknown_020375B8: @ 20375B8
 	.space 0x4
 
 gUnknown_020375BC: @ 20375BC
```
Comment (Diegoisawesome) [inline sym_ewram.txt]: Remove this label, it's no longer needed

**Pull request merged**